app/views/accounting/provisioning/ProvisioningEntries.html

```diff
@@ -14,6 +15,20 @@ <h4></h4>
                 | translate}}</a>
             <br>
             <hr/>
+            
+            <div class="col-md-6">
+                        <div class="form-group">
+                            <input type="text" id="filterDate" name="filterDate" ng-model="filterDate" placeholder="Search By Created Date" 
+                            class="form-control" datepicker-pop="dd MMMM yyyy" is-open="opened1"/>
```

Comment:
Can you name it properly alphanumeric should be avoided, and I dont see its reference in its controller